ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -106,6 +124,10 @@ PPWM:
   guid: "aab62bba-a92f-47bf-bf14-dcb8e61ac7a8"
   description: "Phase power monitoring"
   is_abstract: true
+  opt_uses:
```

Comment:
Please create a new type for these power fields (see `METERS/PLPM` for a good example)

Reply:
Do you want me to create a new type for just `- phase1_power_sensor`,  `- phase2_power_sensor`,  `- phase3_power_sensor`?

Or remove these from PPWM and make a copy of PLPM here?

The PLPM has the apparant power and the power fields in the meters

Reply:
Please create a new type in this file called `PLPM` and it only needs to have the three power_sensor fields as required (you can put the apparent_power fields and the `line_frequency_sensor` in the opt_uses section on the `PLPM` type).